src/actions/index.js

```diff
@@ -0,0 +1,12 @@
+import axios from 'axios';
+
+export const FETCH_WEATHER = 'FETCH_WEATHER';
+
+const ROOT_URL = 'http://api.openweathermap.org/data/2.5/forecast?q=';
+
+export const fetchWeather = async (city) =>
```

Comment:
This function works (in that it returns an action) but you want to use Redux Promise or something like it to simplify implementation.